src/ec/packages/ec-component-blockquote/blockquote.story.js

```diff
@@ -8,17 +9,49 @@ import data from '@ecl/ec-specs-blockquote/demo/data';
 import blockquote from './ecl-blockquote.html.twig';
 import notes from './README.md';
 
+// Labels for the group ids
+const optionalGroupId = 'Optional elements';
+const requiredGroupId = 'Mandatory elements';
+const useCasesGroup = 'Use cases';
+
+// Preserve the adapted specs.
+const defaultData = { ...data };
+
+// declare your toggled data
+const authorBtnToggler = () => {
+  defaultData.author = defaultData.author ? false : data.author;
+};
+
+const formatQuote = passeddata => {
+  // declare your knobs when you format your data
+  const citation = passeddata.citation
+    ? text('Citation', passeddata.citation, requiredGroupId)
+    : false;
+
+  const author = passeddata.author
+    ? text('Author', passeddata.author, optionalGroupId)
+    : false;
+
+  const newQuote = {
+    author,
+    citation,
+  };
+
+  // Return the full object.
+  return newQuote;
+};
+
 storiesOf('Components/Blockquote', module)
   .addDecorator(withKnobs)
   .addDecorator(withCode)
   .addDecorator(withNotes)
   .add(
     'default',
-    () =>
-      blockquote({
-        author: text('Author name', data.author),
-        citation: text('Citation', data.citation),
-      }),
+    () => {
+      button('show/ hide author', authorBtnToggler, useCasesGroup);
```

Comment:
I think we should try to be consistent in labelling these buttons, i'm not that happy with the "with or without" but if we want to change it we have to do it consistently. Anyway, this button in particular has to be removed, the author is not an optional element, as seen before.